src/common.jl

```diff
@@ -336,6 +333,15 @@ function assign_reference_buses(
     return bus_groups
 end
 
+function assign_reference_buses!(
+    subnetworks::Dict{Int, Set{Int}},
+    ref_bus_positions::Set{Int},
+    bus_lookup::Dict{Int, Int},
+)
+    ref_buses = [k for (k, v) in bus_lookup if v in ref_bus_positions]
+    assign_reference_buses!(subnetworks, ref_buses)
```

Comment:
```suggestion
    return assign_reference_buses!(subnetworks, ref_buses)
```

A [style thing](https://github.com/invenia/BlueStyle#method-definitions) to make it clear we care about the return value

Reply:
sorry, I forgot this one